src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -1730,10 +1786,11 @@ static void ending(XmlWriter& xml, Volta* v, bool left)
     }
     QString voltaXml = QString("ending number=\"%1\" type=\"%2\"").arg(number, type);
     voltaXml += ExportMusicXml::positioningAttributes(v, left);
-    if (!v->visible()) {
-        voltaXml += " print-object=\"no\"";
-    }
     if (left) {
+        if (!v->visible()) {
+            voltaXml += " print-object=\"no\"";
+        }
+        voltaXml += color2xml(v);
```

Comment:
Maybe put this into an else branch? Ccolored but invisble doesn't make too much sense, does it?

Reply:
Well, it can be invisible BUT colored. Otherwise we would lose information.

Reply:
True, so I rest my case ;-)